From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Bastiaan van der Plaat <bastiaan.v.d.plaat@gmail.com>
Date: Tue, 6 Oct 2026 19:00:00 +0200
Subject: [PATCH] Skip the WAD checksum only netgames use

The SHA-1 of the WAD directory is only compared by the network client,
which isn't built, like the dehacked checksum next to it. Saves hashing
the directory at startup and the SHA-1 code.
---
 doomgeneric/d_net.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/doomgeneric/d_net.c b/doomgeneric/d_net.c
index b8962cc..2006a66 100644
--- a/doomgeneric/d_net.c
+++ b/doomgeneric/d_net.c
@@ -201,9 +201,8 @@ static void InitConnectData(net_connect_data_t *connect_data)
 
     // Read checksums of our WAD directory and dehacked information
 
-    W_Checksum(connect_data->wad_sha1sum);
-
 #if ORIGCODE
+    W_Checksum(connect_data->wad_sha1sum);
     DEH_Checksum(connect_data->deh_sha1sum);
 #endif
 
